span-processing-config-service-impl/src/main/java/org/hypertrace/span/processing/config/service/validation/SpanProcessingConfigRequestValidator.java

```diff
@@ -39,6 +40,13 @@ public void validateOrThrow(
   public void validateOrThrow(RequestContext requestContext, CreateExcludeSpanRuleRequest request) {
     validateRequestContextOrThrow(requestContext);
     this.validateData(request.getRuleInfo());
+    if (RULE_TYPE_SYSTEM.equals(request.getRuleInfo().getType())) {
```

Comment:
We should also add a validation for deleting system rules

Reply:
Had added the validation for deleting system exclude rules as part of previous PR(https://github.com/hypertrace/config-service/pull/133)
https://github.com/hypertrace/config-service/blob/f786710bded31e03f1de576fc719d6a33b189887/span-processing-config-service-impl/src/main/java/org/hypertrace/span/processing/config/service/SpanProcessingConfigServiceImpl.java#L195.